src/app.ts

```diff
@@ -111,6 +113,8 @@ const switchProfile = async (name?: string, forceMFA?: boolean): Promise<void> =
       selectedProfile.awsOutputFormat
     );
   }
+
+  console.log(chalk.green(`Switched to profile ${selectedProfile.profileName}`));
```

Comment:
We removed this before because inquirer shows which profile you selected so I don't think this is necessary.

Reply:
sure, I changed it so it'll only print this if you passed in the profile as an argument, otherwise in that case there is no feedback at all.

Reply:
Right, that makes sense.